### ISSUE TITLE
add amenity=fast_food

### ISSUE THREAD

**Issue by Discostu36** [babykarte/babykarte.github.io#117]
I think we should put amenity=fast_food on the map to cover for example McDonald's restaurants. I'd like to get your feedback if we should include these into the restaurant filter or if you think this should be a separate filter.

**Comment by ValorNaram** [comment]
fast food is not a restaurant. A restaurant has good ambiente and looks good. A fast food is just for eating and not ambiente.

But McDonalds are usually very baby friendly with some exceptions on the countryside. We can add them.

**Comment by Discostu36** [comment]
Fo you mean only McDonald's?

**Comment by ValorNaram** [comment]
No. Fast food in general. I just used your example. Fast Food comes in the category 'eat' so our rating will be applied to them plus the baby friendly tab specifications for the category `eat`.

See the `applyfor` for each OSM key in `js/overpass.js' and then look at the address `category` from each filter in `js/filters.js` in order to see the connection.

**Comment by carstenhag** [comment]
Just noticed my favorite kebap shop is missing on the map, even though I mapped stroller stuff :D

**Comment by ValorNaram** [comment]
Don't worry! Once we can close #120 I can spend myself on other issues.
